neqo-transport/src/cc/cubic.rs

```diff
@@ -139,7 +139,13 @@ impl WindowAdjustment for Cubic {
 
         let time_ca = self
             .ca_epoch_start
-            .map_or(min_rtt, |t| now + min_rtt - t)
+            .map_or(min_rtt, |t| {
+                if now + min_rtt < t {
+                    min_rtt
```

Comment:
It might be worth commenting here that this only happens when processing old packets that were saved and replayed with old timestamps.  In that case, using a fixed value is safe.

It's a shame that we can't use `saturating_sub()` and all that with `Duration` yet.  Those could make this slightly nicer.